doc/ethapp.asc

```diff
@@ -473,6 +473,8 @@ The following standard Status Words are returned for all APDUs - some specific S
 [width="80%"]
 |===============================================================================================
 | *SW*     | *Description*
+|   6501   | TransactionType not supported
+|   6502   | Not enough space for conversion from uint32_t to string

```

Comment:
that's very accurate yet not so clear
maybe "output buffer too small for u32_to_str input" instead?